src/feed-list.scss

```diff
@@ -0,0 +1,94 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  width: fit-content;

```

Comment:
fit-content will not work in IE. https://caniuse.com/?search=fit-content

Reply:
Done